### PR TITLE
Health Analyzers warn you if the patient is too hot.

### PR DESCRIPTION
**Pull request by kafovofa**
too_hot
The temperature reading turns red, it already did this for low temperatures.

### CODE REVIEW

File: code/game/objects/items/devices/scanners.dm [hunk 1]
```diff
@@ -159,7 +159,7 @@ proc/healthanalyze(mob/living/M as mob, mob/living/user as mob, var/mode = 0, va
 			to_chat(user, {"<span class='notice'>Analyzing Results for the floor:<br>Overall Status: Healthy</span>
 Key: <span class='notice'>Suffocation</span>/<font color='green'>Toxin</font>/<font color='#FFA500'>Burns</font>/<span class='red'>Brute</span>
 Damage Specifics: <span class='notice'>0</span> - <font color='green'>0</font> - <font color='#FFA500'>0</font> - <span class='red'>0</span>
-[(M.undergoing_hypothermia()) ?  "<span class='warning'>" : "<span class='notice'>"]Body Temperature: ???&deg;C (???&deg;F)</span>
+[(M.undergoing_hypothermia() || (M.undergoing_hyperthermia)) ?  "<span class='warning'>" : "<span class='notice'>"]Body Temperature: ???&deg;C (???&deg;F)</span>
```
Comment (help-maint) [inline code/game/objects/items/devices/scanners.dm]: I don't understand how this PR compiles. undergoing_hyperthermia is a proc, not a variable, so shouldn't this be a syntax error?

Comment (kafovofa) [reply code/game/objects/items/devices/scanners.dm]: that is very odd and not how that is meant to be

Comment (kafovofa) [reply code/game/objects/items/devices/scanners.dm]: should be fixed

**Comment by Eneocho** [comment]
I could've sworn they already did that.

**Comment by kafovofa** [comment]
I think I've fucked this up with github mismanagement, going to close and redo it properly

**Pull request closed without merging**